draft-ietf-doh-dns-over-https-latest.mkd

```diff
@@ -277,7 +277,13 @@ accept = application/dns-udpwireformat
 An HTTP response with a 2xx status code ({{RFC7231}} Section 6.3)
 indicates a valid DNS response to the query made in the HTTP
 request. A valid DNS response includes both success and failure
-responses.
+responses. For example, a DNS failure response such as SERVFAIL or
+NXDOMAIN will contain a successful HTTP 2xx status code even though
```

Comment:
Nit: The NXDOMAIN/SERVFAIL won't contain the status code.

Reply:
How dreadfully embarrassing for me.  LPT: don't review PRs during meetings